src/env_zephyr.c

```diff
@@ -9,6 +9,9 @@ bool rcutils_set_env(const char * env_name, const char * env_value)
 
 const char* rcutils_get_env(const char * env_name, const char ** env_value)
 {
+	if (NULL == *env_value) {
```

Comment:
Oh, crap.  The function returns `NULL` for success?!?  That was unexpected. If we were not targeting embedded system, we would return "Unsupported" or "Not implemented".  But having strings to indicate errors is a non-sense on system with tight memory.

Thank you for fixing this!